### PR TITLE
fix P000025 definition

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
see discussion in #157

File: properties/P000025.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+slug: sigma-neighborhood-compact
+name: "$\\sigma$-Neighborhood Compact"
+aliases:
+  - "$\\sigma$-Locally Compact"
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-A space is $\sigma$-locally compact if and only if it is $\sigma$-compact and locally compact.
+A space that is [$\\sigma$-compact](/properties/P000017.md) and [weakly locally compact](/properties/P000023.md).
```
Comment (prabau) [inline properties/P000025.md:11]: I think it should accept the syntax {P17}, {P23} with single braces.  See for example the last line in P000024.md.

File: properties/P000025.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
 ---
 uid: P000025
-slug: sigma-locally-compact
-name: "$\\sigma$-Locally Compact"
+slug: exhaustible-by-compacts
+name: Exhaustable by compacts
```
Comment (prabau) [inline properties/P000025.md:4]: typo: Exhaustible

File: properties/P000025.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+aliases:
+  - "$\\sigma$-Locally Compact"
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-A space is $\sigma$-locally compact if and only if it is $\sigma$-compact and locally compact.
+A space that is {P17} and {P23}.
+Equivalently, $X=\bigcup_{n<\omega}K_n$ where for each $x\in X$, there exists
```
Comment (prabau) [inline properties/P000025.md:12]: It should be specified that each $K_n$ is compact.

File: properties/P000025.md [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
+$n<\omega$ such that $K_n$ is a compact neighborhood of $x$. Also equivalently,
+$X=\bigcup_{n<\omega}K_n$ for which $K_n\subseteq int(K_{n+1})$.
```
Comment (prabau) [inline properties/P000025.md:14]: Also specify that each $K_n$ is compact.

File: properties/P000025.md [hunk 5]
```diff
@@ -1,11 +1,16 @@
...
-slug: sigma-locally-compact
-name: "$\\sigma$-Locally Compact"
+slug: exhaustible-by-compacts
+name: Exhaustable by compacts
+aliases:
+  - "$\\sigma$-Locally Compact"
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
```
Comment (prabau) [inline properties/P000025.md:10]: Could we also add https://en.wikipedia.org/wiki/Exhaustion_by_compact_subsets as a reference, to better justify the name?

File: properties/P000025.md (hunk 3 above)
Comment (StevenClontz) [reply properties/P000025.md:14]: already specified on the next line.

**Review by prabau — APPROVED** [review]

**Pull request merged**